### ISSUE TITLE
twisted.conch fails to adhere to Twisted Coding Standards

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7531]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7531|
|Type|enhancement|
|Created|2014-06-27 13:28:56Z|
|Branch|https://github.com/twisted/twisted/tree/conch-style-7531|

Examples of this include:

* Use of `implements` in place of `implementer`
* Incorrect number of empty lines
* implicit relative imports
* trailing whitespace

The first and third of these impact py3 porting.

<details><summary>Searchable metadata</summary>

```
trac-id__7531 7531
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_conch_style_7531 branches/conch-style-7531
branch_author__lvh lvh
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1403875736936840 1403875736936840
changetime__1403883922656547 1403883922656547
version__None None
owner__lvh lvh

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex commented|
|-|-|

This is resolved by https://github.com/twisted/twisted/pull/31

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)|@lvh set owner to @lvh|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_3"></a>|@lvh commented|
|-|-|

(In [42826]) Branching to conch-style-7531.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_4"></a>|@lvh commented|
|-|-|

Thanks for working on this, Alex! LGTM, failures are unrelated and known. Merging.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_5"></a>|@lvh set status to `closed`|
|-|-|

(In [42829]) Merge conch-style-7531: Minor conch style fixes

Author: Alex
Reviewer: lvh
Closes: [#7531](https://github.com/twisted/twisted/issues/7531)

This makes some minor style fixes to Conch.
